securedrop/tests/test_journalist.py

```diff
@@ -2621,7 +2621,7 @@ def test_passphrase_migration_on_verification(journalist_app):
     assert journalist.pw_salt is None
     assert journalist.pw_hash is None
 
-    # check that that a verification post-migration works
+    # check that a verification post-migration works
```

Comment:
`that` is more often intentionally doubled than other words, but I don't think it needs to be here...